core/src/repair_service.rs

```diff
@@ -567,103 +537,52 @@ impl RepairService {
         new_duplicate_slots: &[(Slot, BankId)],
```

Comment:
How is this function going to be used now? I saw that y ou removed the commented call in the run loop

Reply:
heh, yeah its a skeleton of it's former self right. In the future I think it will be used to monitor a channel between replay and this repair service thread for on demand duplicate repairs.